### DISCUSSION TITLE
비밀번호 암호화하기 (v)

### DISCUSSION THREAD

**Discussion by un0211** [Part3-Team15/taskify#88] [Ideas]
저희 사이트에 중요한 비밀번호로 가입하는 분은 없겠지만, 안전을 위해 비밀번호를 암호화 하는것 어떻게 생각하세요??
제가 스웨거에 테스트해보니 비밀번호 아주 길게 해도 잘 가더라구요!
- 회원가입 시에 원본 비밀번호를 해시함수 이용해서 암호화 한 뒤에 저장
- 로그인 할때도 암호화 한 비밀번호를 전송해서 암호화 한 비밀번호끼리 비교

-> 비밀번호가 자체로 외부에 노출되는 일 X

**Comment by wayandway** [comment]
맞아요! 회원가입 및 로그인 api 연동될 때 비밀번호 암호화할 수 있는 라이브러리나 로직 추가하면 좋겠네요👍
이슈로 등록하면 좋을 것 같아요!
